asv/plugins/git.py

```diff
@@ -98,7 +98,7 @@ def get_date(self, hash):
     def get_hashes_from_range(self, range_spec):
         args = ['log', '--quiet', '--first-parent', '--format=format:%H']
         if range_spec != "":
-            args += [range_spec]
+            args += range_spec.split()
```

Comment:
Oh, I have been using this to run a single commit with asv, like `asv run "-1 <hash>"`, since git log will take more than one argument.